gammapy/datasets/core.py

```diff
@@ -178,6 +183,8 @@ def models(self, models):
         Duplicate model objects have been removed.
```

Comment:
For me, the docstring is a bit obscure... The setter is about unique models: what is a unique model? The docstring describes in fact what does the setter and the input parameter is not precised. Can you enlight a bit this docstring?

Reply:
The setter just display the same output as its associated property. I think we have the same behaviour everywhere. Probably better not change too much docstring in a bugfix that has to beck backported.